streaming/kafka/addons.tf

```diff
@@ -169,85 +114,115 @@ provisioner: ebs.csi.aws.com
 # NOTE: Kafka Zookeeper and Broker pod creation may to 2 to 3 mins
 #---------------------------------------------------------------
 resource "kubectl_manifest" "kafka_namespace" {
   yaml_body  = file("./kafka-manifests/kafka-ns.yml")
-  depends_on = [module.eks_blueprints.eks_cluster_id]
+  depends_on = [module.eks.cluster_id]
 }
 
 resource "kubectl_manifest" "kafka_cluster" {
   yaml_body  = file("./kafka-manifests/kafka-cluster.yml")
-  depends_on = [kubectl_manifest.kafka_namespace, module.eks_blueprints_kubernetes_addons]
+  depends_on = [helm_release.strimzi_operator]
 }
 
 resource "kubectl_manifest" "kafka_metric_config" {
   yaml_body  = file("./kafka-manifests/kafka-metrics-configmap.yml")
-  depends_on = [kubectl_manifest.kafka_cluster]
+  depends_on = [helm_release.strimzi_operator]
 }
 
 #---------------------------------------------------------------
-# Grafana Dashboard for Kafka
-# Login to Grafana dashboard
-#1/ kubectl port-forward svc/grafana-service 3000:3000 -n grafana
-#2/ Admin password: kubectl get secrets/grafana-admin-credentials --template={{.data.GF_SECURITY_ADMIN_PASSWORD}} -n grafana | base64 -D
-#3/ Admin user: admin
+# Install Kafka Montoring Stack with Prometheus and Grafana
+# 1- Grafana port-forward `kubectl port-forward svc/prometheus-grafana 8080:80 -n strimzi-kafka-operator`
+# 2- Grafana Admin user: admin
 #---------------------------------------------------------------
-resource "helm_release" "grafana_operator" {
-  name             = "grafana-operator"
-  repository       = "https://charts.bitnami.com/bitnami"
-  chart            = "grafana-operator"
-  version          = "2.7.8"
-  namespace        = "grafana"
-  create_namespace = true
 
-  depends_on = [module.eks_blueprints.eks_cluster_id]
-}
-
-resource "kubectl_manifest" "grafana_prometheus_datasource" {
-  yaml_body = file("./grafana-manifests/grafana-operator-datasource-prometheus.yml")
-
-  depends_on = [helm_release.grafana_operator]
-}
-
-resource "kubectl_manifest" "grafana_kafka_dashboard" {
-  yaml_body = file("./grafana-manifests/grafana-operator-dashboard-kafka.yml")
+  resource "helm_release" "kube_prometheus_stack" {
+    name             = "prometheus-grafana"
+    chart            = "kube-prometheus-stack"
+    repository       = "https://prometheus-community.github.io/helm-charts"
+    version          = "43.2.1"    
+    namespace        = local.strimzi_kafka_name
+    create_namespace = true
+    description      = "Kube Prometheus Grafana Stack Operator Chart"
+    # timeout          = 600
+    wait             = true
+    values           = [templatefile("${path.module}/helm-values/prom-grafana-values.yaml", {})]
+    depends_on = [aws_eks_addon.vpc_cni]
+  }
 
-  depends_on = [helm_release.grafana_operator]
-}
+  resource "kubectl_manifest" "podmonitor_cluster_operator_metrics" {
+  yaml_body = file("./monitoring-manifests/podmonitor-cluster-operator-metrics.yml")
+  depends_on = [kubectl_manifest.kafka_namespace]
```

Comment:
nit: formatting. add two spaces. Pre-commit didn't pickup thisone?
same for all the resources below

Reply:
Done